float64 angle
bool good_distance_to_stop
int64 lpos
int64 rpos
